audformat/core/testing.py

```diff
@@ -246,57 +246,66 @@ def create_db(minimal: bool = False) -> Database:
     # Schemes #
     ###########
 
-    db.schemes['string'] = Scheme()
-    db.schemes['int'] = Scheme(
-        dtype=define.DataType.INTEGER, minimum=0, maximum=100,
-    )
+    db.schemes['bool'] = Scheme(dtype=define.DataType.BOOL)
```

Comment:
Here we add a scheme with boolean values to cover this type. Other changes are just cosmetic.